### DISCUSSION TITLE
move files and folders to another path

### DISCUSSION THREAD

**Discussion by Razboinicul** [micropython/micropython#10610] [RP2040 / Pico]
I can't move files and folders to another path. I need some help. My board is a rp pico w.

**Comment by robert-hh** [comment]
Usually you move files by renaming. Do you have an example of what you try and fails with it.

**Reply by Razboinicul** [reply]
Can You please give me a code example?

**Reply by robert-hh** [reply]
See the docs about the os module: https://docs.micropython.org/en/latest/library/os.html?highlight=rename#os.rename

**Reply by Razboinicul** [reply]
Thanks

**Reply by Razboinicul** [reply]
I still get EISDIR error and I don't know why.

**Comment by sosi-deadeye** [comment]
This does not work, if you want to move **a file across different file systems**. In this case you must **copy** the file and then **delete** the old file. There is a minimal implementation in `micropython-lib`: https://github.com/micropython/micropython-lib/blob/master/python-stdlib/shutil/shutil.py

The function `shutil.move` is currently not implemented. The original implementation is a bit complicated.

**Reply by Razboinicul** [reply]
Thanks
